internal/pkg/template/template_functions.go

```diff
@@ -86,6 +86,9 @@ func ToSnakeCaseFunc(s string) string {
 // IncFunc increments an integer value and returns the result.
 func IncFunc(i int) int { return i + 1 }
 
+// BoolValue dereference a boolean pointer.
+func BoolValue(p *bool) bool { return aws.BoolValue(p) }
```

Comment:
Should we instead add a method to Telemetry?

```go
func (t *Telemetry) EnableContainerInsights() bool {
   if t == nil { return false }
   return aws.BoolValue(t.ContainerInsights)
}
```

Reply:
Updated! 

I want the `ClusterSettings` blob to be rendered only when `ContainerInsights` is explicitly set to true/false; that is it shouldn't be in the template if `ContainerInsights` is `nil`. That's why there is a bit of churn here.

See if you like what it is now!